pkg/kubelet/dockertools/manager.go

```diff
@@ -64,6 +64,8 @@ const (
 	// we want to able to consider SRV lookup names like _dns._udp.kube-dns.default.svc to be considered relative.
 	// hence, setting ndots to be 5.
 	ndotsDNSOption = "options ndots:5\n"
+
+	containerNetworkEnvKey = "NETWORK"
```

Comment:
How about providing an api at the kubelet itself that the pods can query?
The kubelet can fill up network info about a pod through the status hook in an organized structure. See #9646.